llvm/lib/Transforms/Instrumentation/SanitizerCoverage.cpp

```diff
@@ -949,7 +949,7 @@ void ModuleSanitizerCoverage::InjectCoverageAtBlock(Function &F, BasicBlock &BB,
   DebugLoc EntryLoc;
   if (IsEntryBB) {
     if (auto SP = F.getSubprogram())
-      EntryLoc = DILocation::get(SP->getContext(), SP->getScopeLine(), 0, SP);
+      EntryLoc = DILocation::get(SP->getContext(), 0, 0, SP);
```

Comment:
IMHO this warrants a comment along the lines of "Use a compile-generated source-location for all coverage calls", to aid future readers